v2/util/docker-cleanup.sh

```diff
@@ -6,9 +6,12 @@ if [[ $(/usr/bin/docker ps -a | grep Exited | awk '{print $1 "\\t" $7 "\\t" $10}
 then
   echo "Starting docker cleanup with a max of: $MAX_DOCKER"
   /usr/bin/docker ps -a | grep Exited | awk '{print $1 "\\t" $7 "\\t" $10}' | grep week | awk '{print $1}' | head -n $MAX_DOCKER | xargs /usr/bin/docker rm -v
-  /usr/bin/docker images -a | grep none | awk '{print $3}' | xargs /usr/bin/docker rmi
-  echo "cleanup complete"
+  echo "container cleanup complete"
 else
-  echo "nothing to cleanup"
+  echo "no containers to cleanup"
 fi
+
+/usr/bin/docker images -a  | awk '{print $3}' | xargs /usr/bin/docker rmi
```

Comment:
so just remove everything that's not being used?
what if docker updates to allow `-f` by default for the `rmi` command? (sorry I'm being paranoid about this)